### ISSUE TITLE
Problem starting Nextcloud Deck on Galaxy S9 running Android

### ISSUE THREAD

**Issue by TerryHewitt** [stefan-niedermann/nextcloud-deck#509]
Just installed Nextcloud Deck on my Galaxy S9 running Android from the google play store.

When I start the application I get a screen which fills the device screen.
I click the button "Choose Account"
Briefly I see a smaller window with "Add account", "OK" and "cancel" on it and within 1 second it disappears.

How do I get that smaller window to stay at the front so I can connect to my server?

Best wishes
Terry

**Comment by desperateCoder** [comment]
Hi @TerryHewitt, thaks for the report! 
This shouldn't happen, the app is supposed to show you the accounts registered in your Nextcloud Files app for Android. 
Please make sure you have your account set up in the Files app.
If this doesn't help either, try the following: https://github.com/stefan-niedermann/nextcloud-deck/issues/503#issuecomment-630324302

If even that won't help, please tell us!

**Comment by stefan-niedermann** [comment]
@TerryHewitt can you give us an update whether you were able to solve the issue with the tips of @desperateCoder ?

**Comment by TerryHewitt** [comment]
Stefan,

 

My apologies, for the delay in replying.

 

I deleted my accounts in the base (files)  Nextcloud application.

 

Then I created account in Deck client and connected to my server

 

The I could add my other accounts to Files.

 

Thanks for the advice. Working without problems now.

Best wishes

 

Terry

 

Professor W T Hewitt

WTH Associates Ltd

 

Telephone: +44 7905 759 003

 

Company registration number:7336226

HMRC VAT Registration number: 998 8457 23

 

Company Registered Address

St George's House  | 215 - 219 Chester Road | Manchester | M15 4JE
